packages/hardware-trezor/src/transformers/additionalWitnessRequests.ts

```diff
@@ -0,0 +1,23 @@
+import * as Trezor from 'trezor-connect';
+import { CardanoKeyConst, GroupedAddress, util } from '@cardano-sdk/key-management';
+import { TrezorTxTransformerContext } from '../types';
+import { isNotNil } from '@cardano-sdk/util';
+import concat from 'lodash/concat';
+import uniq from 'lodash/uniq';
+
+export const createRewardAccountKeyPath = (knownAddress: GroupedAddress) => [
+  util.harden(CardanoKeyConst.PURPOSE),
+  util.harden(CardanoKeyConst.COIN_TYPE),
+  util.harden(knownAddress.accountIndex),
+  util.STAKE_KEY_DERIVATION_PATH.role,
+  util.STAKE_KEY_DERIVATION_PATH.index
+];
```

Comment:
Maybe we should use https://github.com/input-output-hk/cardano-js-sdk/blob/chore/lw-6514-hardware-trezor-package/packages/hardware-trezor/src/transformers/keyPaths.ts#L19 and include stakeKeyPath only if it exists?